components/artist_page_cta/view.coffee

```diff
@@ -1,4 +1,5 @@
 Backbone = require 'backbone'
```

Comment:
The changes here are just about propagating the current query params along to the different links/views.